pkg/migrations/op_create_table_test.go

```diff
@@ -1020,6 +1020,191 @@ func TestCreateTable(t *testing.T) {
 				}, rows)
 			},
 		},
+		{
```

Comment:
We could potentially just keep the first of these tests, which wouldn't require the extension.

Do you think that still provides enough coverage?